Automation_Lesson1/Task3/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string stringInput1 = "";
+            string stringInput2 = "";
+            string stringInput3 = "";
+
+            Console.WriteLine("Input first string: ");
+            stringInput1 = Console.ReadLine();
+            stringInput1 = stringInput1.ToUpper();
```

Comment:
@twinkleyarns Could be simplified to the consecutive calls in one line, like Console.ReadLine().XXX
But we will learn it on the next lesson :)